nixos/modules/services/monitoring/grafana-image-renderer.nix

```diff
@@ -106,9 +106,9 @@ in {
       }
     ];
 
-    services.grafana.extraOptions = mkIf cfg.provisionGrafana {
```

Comment:
Thanks for this PR @KFearsoff. As I've been a bit puzzled by it during the migration, I'd like to suggest to add a `mkRemovedOptionModule` on `services.grafana.extraOptions` with a message showing an example of how to use the new `services.grafana.settings`, especially because this settings is using a hierarchical structure instead of a flat one.

Reply:
Ah, once we've fixed the `extraOptions`, other old options bring an error message showing the hierarchical structure to be used, that's great, too bad the `extraOptions` error came first alone.

Reply:
Sorry, I've just noticed that this had already been reported before in https://github.com/NixOS/nixpkgs/pull/191768#issuecomment-1296773914